install.ps1

```diff
@@ -910,9 +910,9 @@ $configXml.save((Join-Path ${Env:VM_COMMON_DIR} "packages.xml"))
 
 # Custom Start Layout setup
 Write-Host "[+] Checking for custom Start Layout file..."
-$layoutPath = Join-Path ${Env:VM_COMMON_DIR} "CustomStartLayout.xml"
+$layoutPath = Join-Path "C:\Users\Default\AppData\Local\Microsoft\Windows\Shell" "LayoutModification.xml"
```

Comment:
@emtuls 

> This changes the name of our custom start layout to the name that Windows uses when actually implementing the custom layout, which will then be used in (2nd PR https://github.com/mandiant/VM-Packages/pull/1137, from VM-Packages) to store it in the correct location.

I do not see that the new filename is used in the second PR. Am I understanding something wrong?

Reply:
good, I just wanted to confirm it before merging this PR :wink: